tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -1109,7 +1109,9 @@ void SpirvEmitter::doFunctionDecl(const FunctionDecl *decl) {
     info = getOrCreateRichDebugInfo(loc);
 
     auto *source = info->source;
-    auto *parentScope = info->scopeStack.back();
+    // Note that info->scopeStack.back() is a lexical scope of the function
+    // caller.
+    auto *parentScope = info->compilationUnit;
```

Comment:
This will result in **all functions** having the compilation unit as their parent. I think for struct/class member functions, the parent scope should be the struct?

Reply:
I see. ok thanks